content/en/getting_started/observability_pipelines/_index.md

```diff
@@ -0,0 +1,202 @@
+kind: Documentation
+aliases:
+    - /observability_pipelines/quickstart/
+further_reading:
+  - link: "/observability_pipelines/production_deployment_overview/"
+    tag: "Documentation"
+    text: "Production deployment overview for the Observability Pipelines Worker"
+  - link: "https://dtdg.co/d22op"
+    tag: "Learning Center"
+    text: "Safe and Secure Local Processing with Observability Pipelines"
+---
+
+## Overview
+
+The [Observability Pipelines Worker][1] can collect, process, and route logs and metrics from any source to any destination. Using Datadog, you can build and manage all of your Observability Pipelines Worker deployments at scale. 
+
+This guide walks you through deploying the Worker in your common tools cluster and configuring the Datadog Agent to send logs and metrics to the Worker.
+
+{{< img src="getting_started/observability_pipelines/dd-pipeline.png" alt="A diagram of a couple of workload clusters sending their data through the Observability Pipelines aggregator." >}}
+
+## Assumptions
+* You are already using Datadog and want to use Observability Pipelines.
+* Your services are deployed to a Kubernetes cluster in Amazon Elastic Kubernetes Service (EKS), Azure Kubernetes Service (AKS), or Google Kubernetes Engine (GKE).
+* You have administrative access to the cluster(s) where the Observability Pipelines Worker is going to be deployed, as well as to the workloads that are going to be aggregated.
+* You have a common tools or security cluster for your environment to which all other clusters are connected.
+
+# Prerequisites
+Before installing, make sure you have:
+
+* A valid [Datadog API key][2].
+* An Observability Pipelines Configuration ID.
+
+You can generate both of these in [Observability Pipelines][3].
+
+To run the Worker on your Kubernetes nodes, you need a minimum of two nodes with one CPU and 512MB RAM available. Datadog recommends creating a separate node pool for the Workers, which is also the recommended configuration for production deployments.
+
+## Provider-specific requirements
+{{< tabs >}}
+{{% tab "AWS EKS" %}}
+* The [AWS Load Balancer controller][1] is required. If you created your Amazon EKS cluster through the UI, then it is probably already installed.
+* Datadog recommends using Amazon EKS >= 1.16.
+
+[1]: https://docs.aws.amazon.com/eks/latest/userguide/aws-load-balancer-controller.html
+{{% /tab %}}
+{{% tab "Azure AKS" %}}
+There are no specific requirements for Azure AKS.
+{{% /tab %}}
+{{% tab "Google GKE" %}}
+There are no specific requirements for Google GKE.
+{{% /tab %}}
+{{< /tabs >}}
+
+## Installing the Observability Pipelines Worker
+### Download the Helm chart
+
+{{< tabs >}}
+{{% tab "AWS EKS" %}}
+Download the [Helm chart][1] for AWS EKS. 
+
+In the Helm chart, replace the `datadog.apiKey` and `datadog.configKey` values to match your pipeline. Then, install it in your cluster with the following commands:
+
+```
+$ helm repo add datadog https://helm.datadoghq.com
```

Comment:
@maycmlee An idea: What if we drop the $ and use the new terminal command styling instead?

Reply:
Good idea! Updated with new shell code block.